client/src/main/scala/skuber/api/package.scala

```diff
@@ -45,7 +45,26 @@ package object client {
      token: Option[String] = None,
      userName: Option[String] = None,
      password: Option[String] = None
-   )
+   ) {
+     override def toString : String = {
+       var result = new StringBuilder("AuthInfo(")
+       clientCertificate.getOrElse("") match {
+         case Left(cert) => result ++= "clientCertificate=" + cert + " "
+         case Right(cert) => result ++= "clientCertificate=<PEM masked> "
+         case _ =>
```

Comment:
The `_` was needed to match on the `getOrElse("")` of the `Option` (which is `String`)